GrandCentralBoard/App/AppDelegate.swift

```diff
@@ -4,10 +4,8 @@
 //
 
 import UIKit
+import GrandCentralBoardCore
```

Comment:
If it was working before without this import, it is not needed (can't check now).